documentation/adr/template.md

```diff
@@ -0,0 +1,68 @@
+# {short title of solved problem and solution}
+
+- **Status:** {Draft | Proposed | Rejected | Accepted | Deprecated | ... | Superseded By [xxx](yyyymmdd-xxx.md)} <!-- REQUIRED -->
+- **Last Modified:** {YYYY-MM-DD when the decision was last updated} <!-- REQUIRED -->
+- **Related Issue:** [#{issue number}](https://github.com/HHS/grants-api/issues) <!-- RECOMMENDED -->
+- **Deciders:** {list everyone involved in the decision} <!-- OPTIONAL -->
```

Comment:
should this be required?

Reply:
Good call! I updated this in [this commit](https://github.com/HHS/grants-api/pull/15/commits/0e4207784335a5e4e8a6da53ce7fa979b9ea075e)